yogstation/code/game/objects/items/plushes.dm

```diff
@@ -116,6 +116,7 @@
 		user.visible_message("<span class='notice'>[user] attempts to stab [src]!</span>", "<span class='suicide'>The knife bounces off [src]'s back before breaking into millions of pieces... [src] glares at [user]!</span>") // You fucked up now son
 		I.play_tool_sound(src)
 		qdel(I)
+		sleep(3 SECONDS)
```

Comment:
```suggestion
		addtimer(CALLBACK(user, /mob/living/.proc/gib), 3 SECONDS) 
```
Get rid of user.gib() and sleep. those are bad

Reply:
Sure, I can do that.